docs/udf.rst

```diff
@@ -317,7 +317,58 @@ To invoke a UDF like this, the apply_neighborhood method is most suitable:
             {'dimension': 'y', 'value': 128, 'unit': 'px'}
         ], overlap=[])
 
+Inspecting variables within UDF
```

Comment:
This section is still redundant given the existing "Logging from a UDF" section at https://github.com/Open-EO/openeo-python-client/blob/21922f1d8afb1ab9b1588fbfea56640717083e47/docs/udf.rst?plain=1#L616-L645

I'd propose to finetune the existing docs if that is necessary

Reply:
ok you are right, later I will try to finetune the existing ones